docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -0,0 +1,996 @@
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteMany
+  storageClassName: azurefile
+  azureFile:
+    secretName: 0730azure-secret
+    shareName: 0730-weblogic-1597391432
+    readOnly: false
+  mountOptions:
+  - dir_mode=0777
+  - file_mode=0777
+  - uid=1000
+  - gid=1000
+  - mfsymlinks
+  - nobrl
+```
+
+We have provided another configuration file `pvc.yaml` for the PersistentVolumeClaim.  Both `pv.yaml` and `pvc.yaml` have exactly the same content for `storageClassName` attributes. This is required. We set the same value to the `metadata` property in both files. The following content is an example that uses the persistent volume claim name `0730-azurefile-1597391432`.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: 0730-azurefile-1597391432
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: azurefile
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+Use the `kubectl` command to create the persistent volume and persistent volume claim.
+
+```bash
+$ kubectl apply -f ~/azure/weblogic-on-aks/pv.yaml
+$ kubectl apply -f ~/azure/weblogic-on-aks/pvc.yaml
+```
+
+You will see the following output after each command, respectively.
+
+```bash
+$ kubectl apply -f ~/azure/weblogic-on-aks/pv.yaml
+persistentvolume/0730-azurefile-1597391432 created
+$ kubectl apply -f ~/azure/weblogic-on-aks/pvc.yaml
+persistentvolumeclaim/0730-azurefile-1597391432 created
+```
+
+Use the following command to verify:
+
+```bash
+$ kubectl get pv,pvc
+```
+
+Example output:
+
+```bash
+$ kubectl get pv,pvc
+NAME                                          CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                STORAGECLASS   REASON   AGE
+persistentvolume/0730-azurefile-1597391432   5Gi        RWX            Retain           Bound    default/0730-azurefile-1597391432   azurefile               16m
+
+NAME                                               STATUS   VOLUME                       CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/0730-azurefile-1597391432   Bound    0730-azurefile-1597391432   5Gi        RWX            azurefile      16m
+```
+
+> **Note**: Carefully inspect the output and verify it matches the above. `ACCESS MODES`, `CLAIM`, and `STORAGECLASS` are vital.
+
+#### Install WebLogic Server Kubernetes Operator into the AKS cluster
+
+The Oracle WebLogic Server Kubernetes Operator is an adapter to integrate WebLogic Server and Kubernetes, allowing Kubernetes to serve as a container infrastructure hosting WLS instances.  The operator runs as a Kubernetes Pod and stands ready to perform actions related to running WLS on Kubernetes.
+
+Kubernetes Operators use [Helm](https://helm.sh/) to manage Kubernetes applications. Use the configuration file in `~/azure/weblogic-on-aks/cluster-admin-role.yaml` to grant the Helm service account with the permission to install the operator. The following are the contents of `cluster-admin-role.yaml`:
```

Comment:
The instructions here to create a ClusterRoleBinding and to grant the role to a Helm service account are only required for Helm 2.x, which is no longer supported.